install_config/install/advanced_install.adoc

```diff
@@ -752,6 +752,70 @@ openshift_examples_modify_imagestreams=true
 |Set to `true` if pointing to a registry other than the default. Modifies the image stream location to the value of `*oreg_url*`.
 |===
 
+[[advanced-install-configuring-registry-location]]
+=== Configuring the Registry Console
+
+If you are using a Cockpit registry console image other than the default or require a specific version of the console,
+specify the desired registry within the
+*_/etc/ansible/hosts_* file. 
+
+----
+openshift_cockpit_deployer_prefix=<registry-name>/<namespace>/
+openshift_cockpit_deployer_version=<cockpit-image-tag>
+----
+
+.Registry Variables
+[options="header"]
+|===
+
+|Variable |Purpose
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to the directory where the image is located.
+
+|`*openshift_cockpit_deployer_version*`
+|Specify the Cockpit image verion.
+|===
+
+For example:
+If your image is at *_registry.example.com/openshift3/registry-console_* and you require
+version 1.4.1, enter:
+----
+openshift_cockpit_deployer_prefix='registry.example.com/openshift3/' 
+openshift_cockpit_deployer_version='1.4.1'
+----
+
+[[advanced-install-configuring-registry-location]]
```

Comment:
@mburke5678 - this section was cherry picked twice in this branch. So on top of: https://github.com/openshift/openshift-docs/pull/7331/files#r168657923, we have the same anchor id three times in this file :)

Reply:
@vikram-redhat Thank you. It appears you fixed it?

Reply:
@mburke5678 yeah :)